pkgs/development/python-modules/pyairports/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyairports";
+  version = "2.1.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-PWCnJ/zk2oG5xjk+qK4LM9Z7N+zjRN/8hj90njrWK80=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
```

Comment:
`build-system` is preferred to `nativeBuildInputs` for pythonPackages

Reply:
(@cfhammill to clarify, it's preferred for python dependencies, but not for e.g. hooks)